test/rotation_tests.jl

```diff
@@ -209,6 +209,28 @@ all_types = (RotMatrix3, RotMatrix{3}, AngleAxis, RotationVec,
         end
     end
 
+    @testset "Rotation result eltype" begin
```

Comment:
@hyrodium is this sentence clear for you? I'm not a native English speaker.

Reply:
Should be fine! I'm also a non-native English speaker :+1: